code/datums/tips/tips.dm

```diff
@@ -74,10 +74,9 @@ GLOBAL_LIST_EMPTY(tips_by_category)
 		"Just like real life the entropy of the game can only increase with time. If things aren't on fire yet, just wait.",
 		"Cleanbot.",
 		"It's fun to try and predict the round type from the tip of the round message.",
-		"The Quartermaster has finally become a Head of Staff. He only needed to be renamed to Operations Manager. It can be that easy, kids.",
```

Comment:
Why is this being removed?

Reply:
This tip as far as I can tell was added to celebrate the quartermaster becoming the Operations Manager role. While funny at the time, its been a couple years. Now, its just a bit odd to see. So, removed for not really being relevant anymore.